util/dvsim/Testplan.py

```diff
@@ -11,17 +11,48 @@
 from pathlib import Path
 
 import hjson
+import logging as log
 import mistletoe
 from tabulate import tabulate
 
 
+class JobTime:
+    # Possible units.
+    units = ["h", "m", "s", "ms", "us", "ns", "ps", "fs"]
+    dividers = [60.0,] * 3 + [1000.0,] * 5
+
+    def __init__(self):
+        self.time = 0.0
+        self.unit = "s"
+
+    def __str__(self):
+        if self.time == 0:
+            return ""
+
+        try:
+            index = self.units.index(self.unit)
+        except ValueError as e:
+            log.debug("Unsupport JobTime unit: " + self.unit + f"\n{e}")
```

Comment:
Now this would be a bad coding error that should be found and fixed! Perhaps `log.error`?